backend/src/main/java/gov/cdc/usds/simplereport/service/TestOrderService.java

```diff
@@ -559,6 +567,7 @@ public TestEvent correctTest(
         });
     _resultRepo.saveAll(results);
     _testEventReportingService.report(newRemoveEvent);
+    _fhirQueueReportingService.report(newRemoveEvent);
```

Comment:
Same goes here, we'll need to check if the test is actually a multiplex test before putting it on the FHIR queue.